### ISSUE TITLE
Don't display explorer URL separately

### ISSUE THREAD

**Issue by fubuloubu** [ApeWorX/ape#1271]
### What went wrong?

Minor UX nitpick, but when you have an explorer installed Ape adds any extra line to your console output to show you a link to the explorer, below the previous txn hash:

```
INFO: Submitted 0xf099....a6eb
etherscan URL: https://etherscan.io/tx/0xf099....a6eb
```

It would be nicer if it detected if an explorer was installed, and displayed the explorer URL *instead of* the hash (i.e. the hash is the "fallback" if no explorer is installed)

### How can it be fixed?

So make it look more like this with an explorer installed:

```
INFO: Submitted https://etherscan.io/tx/0xf099....a6eb
```

Explorer not installed:
```
INFO: Submitted 0xf099....a6eb
```

**Comment by MadhuMPandurangi** [comment]
hi @fubuloubu, I'm interested to work on this issue and taking this as my college project. Can you please assign me?It'd be very helpful.
thank you

**Comment by fubuloubu** [comment]
> hi @fubuloubu, I'm interested to work on this issue and taking this as my college project. Can you please assign me?It'd be very helpful. thank you

I don't think I can assign you @MadhuMPandurangi but you are more than welcome to create a PR for this issue, just add `fixes: #1271` in the body of your PR and we will know how to review it!

Thank you!

**Comment by unparalleled-js** [comment]
This is a nice request, but I would not label this is a bug.
